tests/unit/keyspace.tcl

```diff
@@ -254,7 +254,7 @@ start_server {tags {"keyspace"}} {
         assert_refcount 1 mynewlist{t}
         r del mylist{t}
         assert_equal $digest [debug_digest_value mynewlist{t}]
-    }
+    } {} {needs:debug}
```

Comment:
added because the `assert_refcount`

Reply:
actually, in this case we use assert_refcount (not OBJECT REFCOUNT directly), so we can modify assert_refcount to skip the check, same as we did in debug_digest_value.

i.e. we didn't mark that test as `debug` when we used DEBUG, it'll be a shame to mark it as such because of REFCOUNT.

the reasoning is that we can increase coverage for such cases, by ignoring some simple check rather than skip the whole test.

Reply:
right... I did see debug_digest_value, and then saw assert_encoding (ignoreencoding), and then messing ignorexxx and debug flag.